code/modules/antagonists/nations/factions/cargo.dm

```diff
@@ -0,0 +1,19 @@
+/datum/team/nationcargo
```

Comment:
team/nations/cargo

All of these teams should be under a subtype specifically for nations